src/tasks/tasks.service.ts

```diff
@@ -1,12 +1,18 @@
 import { Injectable } from '@nestjs/common';
 
 import { Task } from 'src/common/entities/task.entity';
+import { CreateTaskDto } from './tasks.dto';
 import { TaskRepository } from './tasks.repository';
 
 @Injectable()
 export class TasksService {
   constructor(private readonly taskRepository: TaskRepository) {}
+
   async findAll(): Promise<Task[]> {
-    return this.taskRepository.getAllTasks();
+    return await this.taskRepository.findAll();
+  }
+
+  async createTask(createTaskDto: CreateTaskDto): Promise<Task> {
```

Comment:
```suggestion
  async createOne(createTaskDto: CreateTaskDto): Promise<Task> {
```

Don't need task in the method name.